Podfile.lock

```diff
@@ -125,7 +125,7 @@ EXTERNAL SOURCES:
     :branch: signal-master
     :git: https://github.com/WhisperSystems/JSQMessagesViewController.git
   SignalServiceKit:
-    :path: "."
```

Comment:
What version of cocoapods are you using? Can you use the version specified in the Gemfile.lock?

 (or just run `bundle` to install the correct version, then invoke with `bundle exec pod install`)